plugin/vim-gh-line.vim

```diff
@@ -48,14 +48,27 @@ if !exists('g:gh_cgit_url_pattern_sub')
     let g:gh_cgit_url_pattern_sub = []
 endif
 
-func! s:gh_line(action) range
+if !exists('g:git_remote')
+    let g:git_remote = ""
+endif
+
+func! s:gh_line(action, force_interactive) range
     " Get Line Number/s
     let lineNum = line('.')
     let fileName = resolve(expand('%:t'))
     let fileDir = resolve(expand("%:p:h"))
     let cdDir = "cd '" . fileDir . "'; "
 
-    let origin = system(cdDir . "git config --get remote.origin.url")
+    " try to find git remote
+    if a:force_interactive == 1 || g:git_remote == ""
+      let g:git_remote = s:find_git_remote(cdDir)
+    endif
+
+    if g:git_remote == ""
+      echom "vim-gh-line CAN NOT find git remote"
```

Comment:
Should this be a throw ?

Reply:
no, there are cases that it is a git local repo but user didn't notice that, I think we could just keep it.

Reply:
ok makes sense, I did not think about clones from other directories in the same file system.